src/assets/styles/LayoutStyle.ts

```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+    listItemContainer: {
+        margin: 8,
+        padding: 5,
+        justifyContent: 'center',
+        borderRadius: 15,
```

Comment:
@hellochirag please reduce the border radius to have the same value as it was initially on card component

Reply:
@hellochirag there is also conflict in src/screens/InboundOrderList/index.tsx

Reply:
Hello @awalkowiak  there have not an initial value for border Radius property, I have miss to add card style in product screen that's why I have made this change for apply new style

Reply:
@awalkowiak conflict has been resolved

Reply:
@hellochirag the Card component has its own "initial value" for the border which is 4px. By: 

> please reduce the border radius to have the same value as it was initially on card component

I meant to remove this line in the styling and not overwrite the default border-radius, because the `borderRadius: 15` version looks ugly. Please remove it.